xbmc/video/dialogs/GUIDialogVideoManagerVersions.cpp

```diff
@@ -645,7 +645,7 @@ bool CGUIDialogVideoManagerVersions::GetSimilarMovies(const std::shared_ptr<CFil
 }
 
 bool CGUIDialogVideoManagerVersions::AddSimilarMovieAsVersion(
-    const std::shared_ptr<CFileItem> itemMovie)
+    const std::shared_ptr<CFileItem>& itemMovie)
```

Comment:
It would be better to pass this as `const CFileItem& itemMovie`, unless the copy is intentional then it should be `std::shared_ptr<CFileItem> itemMove`. I know it's out of the scope of the PR but given that this is new code this change should be scrutinized.

Reply:
Highlighting for @ksooo and @CrystalP

Reply:
It can be changed without issue and was an oversight when the code was written.

Reply:
Cool, just bringing it to your attention.